.github/dependabot.yaml

```diff
@@ -7,8 +7,9 @@ updates:
     time: "10:00"
   open-pull-requests-limit: 10
   versioning-strategy: increase
-  allow:
-    - dependency-type: "direct"
+  ignore:
+    - dependency-name: '*'
```

Comment:
Do we want to allow this for indirect deps as well or just focus on the 'direct' dependencies?

As using the `*` will mean it does all and just results in the lockfile updating and that change then not making it to the consumer